scripts/cipublish

```diff
@@ -19,7 +19,7 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
     else
         echo "Publishing artifacts to Sonatype"
         if [[ -n "${CIRCLE_TAG}" ]]; then
-            ./sbt "++${SCALA_VERSION:-2.11.12}" publish sonatypeRelease
+            ./sbt ";++${SCALA_VERSION:-2.11.12};sonatypeOpen $(date +%s);publish;sonatypeRelease"
```

Comment:
Using Unix time here was an arbitrary decision since the repository description cannot be left blank.

Open to other ideas.

Reply:
We've used SHA for things like this just to track down specific bits of code, but using the date seems good to me